libs/ngxtension/explicit-effect/README.md

```diff
@@ -0,0 +1,3 @@
+# ngxtension/create-effect
+
+Secondary entry point of `ngxtension`. It can be used by importing from `ngxtension/create-effect`.
```

Comment:
```suggestion
# ngxtension/explicit-effect

Secondary entry point of `ngxtension`. It can be used by importing from `ngxtension/explicit-effect`.
```